ext/opentelemetry-ext-flask/src/opentelemetry/ext/flask_util/__init__.py

```diff
@@ -0,0 +1,93 @@
+# Note: This package is not named "flask" because of
```

Comment:
Not a beautiful solution, but I think you can get around this with a `pylint: disable: https://github.com/dynatrace-oss-contrib/opentelemetry-python/pull/1.

Reply:
As stated in https://github.com/dynatrace-oss-contrib/opentelemetry-python/pull/1#discussion_r339048437, I'd rather not do that. What I could do is rename this to `_flask_impl.py` and then have a `flask.py` that does just `from _flask_impl import *`.

Reply:
Since pylint seems to have no problems with the `flask` submodule now (Was it fixed? Did I use a module instead of a package last time I tried? Who knows...), I changed it to simply `flask`.